site/components/src/main/java/uk/nhs/digital/common/components/catalogue/ApiCatalogueComponent.java

```diff
@@ -0,0 +1,84 @@
+package uk.nhs.digital.common.components.catalogue;
```

Comment:
Some files are showing as new here but really they have just a few changes but have had their package renamed.

Reply:
line 19 has the mapping document path now, there a new request parameter line 48. some other stuff that was in this class is now in CatalogueComponent for inheritance.